lib/hash_ring/ets_config.ex

```diff
@@ -0,0 +1,42 @@
+defmodule HashRing.ETS.Config do
+    use GenServer
+
+    defstruct monitored_pids: %{}
```

Comment:
Why not just make the whole state this?

Reply:
i always like to have the state in gen servers be module struct, because if you want to add more things in the future, you don't have to  touch as much code.

Reply:
+1